app/presenters/vacancy_presenter.rb

```diff
@@ -79,4 +79,30 @@ def contract_type_with_duration
     duration = model.fixed_term? ? "(#{model.contract_type_duration})" : nil
     [type, duration].compact.join(" ")
   end
+
+  private
+
+  def fix_bullet_points(text)
```

Comment:
should this look for asterisks also?

Reply:
Do people use asterisks in this way?

Reply:
Example! 
https://teaching-vacancies.service.gov.uk/jobs/teacher-of-biology-0bfc9e23-4386-414f-9cbd-939ec4fb629c